.github/workflows/ci.yml

```diff
@@ -204,6 +204,20 @@ jobs:
       - name: 'Check working directory status'
         run: './scripts/check-git-status.sh'
 
+  cargo_test:
```

Comment:
nit: can we use `cargo-test` for consistency? Or rename everything with underscores? (Happy with either)

Reply:
Good catch, my brain was in Rust mode!